### ISSUE TITLE
bug: Enabling Apple Clang Module Verifier

### ISSUE THREAD

**Issue by qliqdev** [ionic-team/capacitor#6475]
### Capacitor Version

```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 4.7.3
  @capacitor/core: 4.7.3
  @capacitor/android: 4.7.3
  @capacitor/ios: 4.7.3

Installed Dependencies:

  @capacitor/cli: 4.7.3
  @capacitor/core: 4.7.3
  @capacitor/android: 4.7.3
  @capacitor/ios: 4.7.3

[success] iOS looking great! 👌
[success] Android looking great! 👌

```

### Platform(s)

- [ ] ios

### Current Behavior

Xcode build fails with error
```
double-quoted include "CDVViewController.h" in framework header, expected angle-bracketed instead
double-quoted include "CDVPlugin.h" in framework header, expected angle-bracketed instead
double-quoted include "CDVConfigParser.h" in framework header, expected angle-bracketed instead
......
and all Capacitor ios files
```
where code written like

```
#import "CDVPlugin.h"
#import "CDVConfigParser.h"
#import "CDVCommandDelegate.h"
```

### Expected Behavior

Build should successfully pass

### Other Technical Details

`npm --version` output: `9.2.0`

`node --version` output: `v18.12.1`

`pod --version` output (iOS issues only): `1.12.0`
`XCode version` output (iOS issues only): `Version 14.3 (14E222b)`

### Additional Context

Enabling Apple Clang Module Verifier is Xcode recommendation

<img width="601" alt="image" src="https://user-images.githubusercontent.com/5783161/230373005-805c05ea-f2f1-4380-a141-31d32c1188db.png">
<img width="1252" alt="image" src="https://user-images.githubusercontent.com/5783161/230373941-f7e273e6-c156-4322-9829-38881385408e.png">

**Comment by ohlive** [comment]
Hi, i have the same problem.
Have you find a solution?
thx

**Comment by qliqdev** [comment]
@ohlive No, I'm just skipping Xcode recommendations. It always staying recommeded

**Comment by ohlive** [comment]
To my side, i cant ignore this. I disabled "Quoted include in framework" to xcode settings, my build failed. I don't know what I should do

**Comment by qliqdev** [comment]
<img width="693" alt="image" src="https://github.com/ionic-team/capacitor/assets/5783161/130315fd-f784-4f19-b727-337cf23c39ee">

try to disable it

**Comment by ohlive** [comment]
@qliqdev Really thanks you, its ok now. Im not really familiar to Xcode so thanks again.

I find your recommendation here : 
<img width="671" alt="Capture d’écran 2023-06-15 à 11 14 21" src="https://github.com/ionic-team/capacitor/assets/7570094/a1fb6b7d-f53d-4dee-95df-2a20e85e898f">

**Comment by ohlive** [comment]
Sorry, i see i dont respond. 
You rigth, thx to help me!

**Comment by khaelou** [comment]
Same here when attempting to perform changes, I rebuilt the project and skipped this to bypass the issue

**Comment by mledwards** [comment]
I deleted my `ios` folder, did `npm cap add ios` again, then `npm cap open ios`, ignored the "Recommended setting updates" and it built successfully 🥳

**Comment by vrdriver** [comment]
Yes, I had to do this for all of the Capacitor "targets". Annoying, but it builds and runs again in the emulator.

> @qliqdev Really thanks you, its ok now. Im not really familiar to Xcode so thanks again.
> 
> I find your recommendation here : <img alt="Capture d’écran 2023-06-15 à 11 14 21" width="671" src="https://private-user-images.githubusercontent.com/7570094/246061010-a1fb6b7d-f53d-4dee-95df-2a20e85e898f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM1OTczMTMsIm5iZiI6MTcwMzU5NzAxMywicGF0aCI6Ii83NTcwMDk0LzI0NjA2MTAxMC1hMWZiNmI3ZC1mNTNkLTRkZWUtOTVkZi0yYTIwZTg1ZTg5OGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMjZUMTMyMzMzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmVmY2ZkZjQ0MTQxZmU3ZTFjY2UwMTIxMGM4YjUzNWM2YTc1NmE3ODhjMDNhMmZlMmNlOGI3YTBiMTM0ZTM1YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lyOCUiuqeXGy3U_fF0971OdX7upaFAm1j6qMRre5_QM">

**Comment by eric-horodyski** [comment]
Closing as part of our backlog cleanup effort to keep Capacitor focused on current, actionable work.

More context: [https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant in the latest version, please comment with updated details and we can take another look.
